spiffworkflow-frontend/test/browser/process_groups/test_can_perform_crud_operations.py

```
@@ -0,0 +1,139 @@
+    count = delete_btns.count()
+    found = False
+    for i in range(count):
+        btn = delete_btns.nth(i)
+        if btn.is_visible():
+            btn.click()
+            found = True
+            break
+    if not found:
+        print("[DEBUG] Could not find visible Delete button in dialog!")
+        print_page_details(page)
+        assert False, 'Delete button for modal not found'
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)